src/shared/client/cg_api.cpp

```diff
@@ -548,9 +558,12 @@ void trap_R_GetTextureSize( qhandle_t handle, int *x, int *y )
 
 qhandle_t trap_R_GenerateTexture( const byte *data, int x, int y )
 {
+	size_t size = x * y * 4;
```

Comment:
I think we should assert on generating a zero l/w texture. Seems like something we shouldn't want?

Reply:
Done.